libs/network/src/muddle/router.cpp

```diff
@@ -268,6 +268,16 @@ void Router::Route(Handle handle, PacketPtr packet)
   LOG_STACK_TRACE_POINT;
   FETCH_LOG_DEBUG(LOGGING_NAME, "Routing packet: ", DescribePacket(*packet));
 
+  // probe command, bypass all checks
```

Comment:
Attack vector? Wouldn't this allow someone malicious to swamp another node's muddle using the probe channel, as their messages would bypass checks?